examples/hooks/App.tsx

```diff
@@ -81,6 +82,17 @@ export function App() {
               <RangeInput attribute="price" />
             </Panel>
           </DynamicWidgets>
+          <Panel header="NumericMenu">
+            <NumericMenu
+              attribute="price"
+              items={[
+                { label: 'All' },
+                { label: 'Less than 500$', end: 500 },
```

Comment:
This is nitpicking, but I'd place the $ currency before the value.

Reply:
You're right. I copied the example from the docs but your suggestion is more correct.

Reply:
See db6e4e1a8e93781e8d33b9e2a32c0b6326417344.